app/helpers/bookings_helper.rb

```diff
@@ -1,7 +1,74 @@
 module BookingsHelper
 
-  def fetch_booking_info
+  def update_saved(booking_params)
+    if @booking.update(booking_params)
+        @booking.save
+        message = "Update successful."
+    else
+      message = "Update failed."
+    end
+  end
+
+  def booking_params
+    params.require(:booking).
+    permit( :no_of_passengers,
```

Comment:
Use 2 (not 0) spaces for indenting an expression spanning multiple lines.<br>Space inside parentheses detected.